cypress/common/selectors.ts

```diff
@@ -100,14 +103,25 @@ export const buttons = {
     nativeFederationReactButtons: {
         hostButton: '[data-e2e="HOST_BUTTON"]',
         remoteButton: '[data-e2e="REMOTE_BUTTON"]'
-    }
+    },
+    actionButton: 'action-button',
+    closeButton: '.closebtn'
 }
 
 export const alertMessages = {
-    angularAlertMessage: '.alert-danger'
+    angularAlertMessage: '.alert-danger',
+    alertBox: 'alert-box',
+    alert: '.alert'
 }
 
 export const block = {
+    comprehensiveDemoBlockFirstApp3: '.makeStyles-root-1',
```

Comment:
you can merge comprehensiveDemoBlock selectors to object

Reply:
Fixed